### ISSUE TITLE
Override on tank heating

### ISSUE THREAD

**Issue by Klisterkongen** [cjaliaga/home-assistant-aquarea#54]
Override on heating tank does not seem to work unfortunantly

![image](https://github.com/cjaliaga/home-assistant-aquarea/assets/97479658/cd053420-a10e-4f32-bd26-9d79efa1454c)

**Comment by cjaliaga** [comment]
@Klisterkongen could you give me more details about what's the expectation and what's the behavior you're seeing?

**Comment by Klisterkongen** [comment]
> @Klisterkongen could you give me more details about what's the expectation and what's the behavior you're seeing?

Yes, and thank you very much for your response. When clicking “Heating”, I would hope this would override the heat pump’s current activity, and begin to prioritize heating the tank and not the floor heating of the house. Instead, it shows “Heating” for a few seconds, and then changes back to “Mode”.

![image](https://github.com/cjaliaga/home-assistant-aquarea/assets/97479658/fdae2717-10bb-4458-baf9-38fec59c50f1)

**Comment by adamus-tork** [comment]
> Instead, it shows “Heating” for a few seconds, and then changes back to “Mode”.

The Panaosic Cloud service is not reliable. Even their app does glitching.

> I would hope this would override the heat pump’s current activity

I think we would need force DHW|heater action for this.

The guy from Panasic told me that there is a setting which prevents the indoor unit to switch from X to Y for n period of time. Like it will heat the house for 1,5 hours then checks the hot water tank, and only changes back to heating when its reached the heat.

... theoretically, since I had to manually switch to hot water in the last 2 days from heating even when it saw that that was below -12˚C from the setting.

**Comment by Klisterkongen** [comment]
> > Instead, it shows “Heating” for a few seconds, and then changes back to “Mode”.
> 
> The Panaosic Cloud service is not reliable. Even their app does glitching.
> 
> > I would hope this would override the heat pump’s current activity
> 
> I think we would need force DHW|heater action for this.
> 
> The guy from Panasic told me that there is a setting which prevents the indoor unit to switch from X to Y for n period of time. Like it will heat the house for 1,5 hours then checks the hot water tank, and only changes back to heating when the water tank reached the goal.
> 
> ... theoretically, since I had to manually switch to hot water in the last 2 days from heating even when it saw that that was below -12˚C from the setting.

A few months ago, the web interface at panasonic smart cloud was updated with Force DHW and "Powerfull", -i would LOVE these to be avaliable in the HA integration!

![image](https://github.com/cjaliaga/home-assistant-aquarea/assets/97479658/9b4bd1a0-b1dd-4266-a7ac-79989f2e2a1b)

**Comment by cjaliaga** [comment]
I'm working on adding Force DHW. I'm wondering tho what would be the best way: setting it as a mode, a switch entity that you can turn on and off...

Any suggestion?
